app/models/reservation.rb

```diff
@@ -414,7 +419,16 @@ def update(current_user, new_params, new_notes) # rubocop:disable all
             name = 'Item'
             old_val = diff[0] ? EquipmentItem.find(diff[0]).md_link : 'nil'
             new_val = diff[1] ? EquipmentItem.find(diff[1]).md_link : 'nil'
+          when 'checked_out'
+            name = 'Checkout Time'
+            old_val = diff[0] ? diff[0].to_s(:long) : 'nil'
+            new_val = diff[1] ? diff[1].to_s(:long) : 'nil'
+          when 'checked_in'
+            name = 'Checkin Time'
+            old_val = diff[0] ? diff[0].to_s(:long) : 'nil'
+            new_val = diff[1] ? diff[1].to_s(:long) : 'nil'
           end
+          update_overdue
```

Comment:
Let me see if I get this right, should I remove the condition or should I add an additional call back with a different condition?